README.md

```diff
@@ -478,6 +499,107 @@ service.getDevices().then((resp) => {
   // err - exception message object or status code
 });
 ```
+<a name="Service+getDevicesEntries"></a>
+
+### service.getDevicesEntries() ⇒ <code>Promise</code>
+Sends request to get all registered devices entries.
+
+**Kind**: instance method of [<code>Service</code>](#Service)  
+**Returns**: <code>Promise</code> - Promise with a list of endpoints  
```

Comment:
I think `Promise with a list of device entries` would be more uniform throughout the readme